test/tests/ver-1ka/comparison_ver-1ka.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
+import pandas as pd
+import os
+from matplotlib import gridspec
+import matplotlib.pyplot as plt
+
+# Changes working directory to script directory (for consistent MooseDocs usage)
+script_folder = os.path.dirname(__file__)
+os.chdir(script_folder)
+
+# Constants
+S = 1e20 # Source term (m^-3 * s^-1)
+V = 1 # Volume (m^3)
+kb = 1.380649e-23 # Boltzmann constant (J/K)
+T = 500 # Temperature (K)
+
+# Extract time and pressure data
+if "/TMAP8/doc/" in script_folder:     # if in documentation folder
+    csv_folder = "../../../../test/tests/val-1ka/gold/ver-1ka_out.csv"
+else:                                  # if in test folder
+    csv_folder = "./gold/ver-1ka_out.csv"
+expt_data = pd.read_csv(csv_folder)
+TMAP8_time = expt_data['time']
+TMAP8_pressure = expt_data['v']
```

Comment:
```suggestion
TMAP8_pressure = expt_data['pressure']
```
That is what is currently causing the failures in the tests.